src/test/scala/ai/privado/languageEngine/java/tagger/collection/CollectionUtilityTest.scala

```diff
@@ -188,7 +149,6 @@ class CollectionUtilityTest extends JavaFrontendTestSuite {
       val jsonOutput  = cpg.getPrivadoJson()
       val ingressUrls = jsonOutput(Constants.ingressUrls).asArray.get.toList
 
-      ingressUrls.size shouldBe 7
       ingressUrls.map(_.noSpaces).contains("\"/api/public/user/login\"") shouldBe true
```

Comment:
why have we removed this check?

Reply:
I was changing it to list test instead of size. Will add in next commit.